src/parser/cssParser.ts

```diff
@@ -402,6 +402,40 @@ export class Parser {
 		return this.finish(node);
 	}
 
+	// parsing a list of selectors as pseudo selector arguments
+	// e.g. :matches(div, span)
+	public _tryParseSelectorList(): nodes.Node {
+		let mark = this.mark();
+
+		if (this._parseSelector(false)) {
+			while (this.accept(TokenType.Comma) && this._parseSelector(false)) {
+				// loop
+			}
+			if (this.accept(TokenType.ParenthesisR)) {
+				this.restoreAtMark(mark);
+				return this._parseSelectorList();
+			}
+		}
+		this.restoreAtMark(mark);
+		return null;
+	}
+
+	public _parseSelectorList(): nodes.Node {
```

Comment:
I modeled my change after `_parseRuleSet`.
Wonder if returning `nodes.Node` is correct, though. Thinking I should let it return `nodes.Node[]` and use `this.addChildren` in `_parsePseudo`.